attribute-service/build.gradle.kts

```diff
@@ -102,7 +103,9 @@ tasks.jacocoIntegrationTestReport {
 hypertraceDocker {
   defaultImage {
     javaApplication {
-      port.set(9012)
+      ports.add(9012)
+      adminPort.set(9013)
+      healthCheck.set("HEALTHCHECK --interval=2s --start-period=15s --timeout=2s CMD /bin/grpc_health_probe -addr=:9012 -connect-timeout=100ms -rpc-timeout=150ms")
```

Comment:
don't love this, but our base plugin doesn't and shouldn't assume a grpc server for its default health check.